desc/objectives/normalization.py

```diff
@@ -21,6 +19,12 @@ def compute_scaling_factors(thing):
 
         scales["R0"] = R00
         scales["a"] = np.sqrt(np.abs(R10 * Z10))
+        # TODO: also implement for FourierRZToroidalSurface,
+        # need to add "A" compute fxn for that FourierRZToroidalSurface
+        if np.isclose(scales["a"], 0):
+            # R10 and Z10 can be 0 for stellarator asymmetric equilibria,
+            # just use the computed minor radius in this case
+            scales["a"] = thing.compute("a")["a"]
```

Comment:
What if we always did:
`scales["a"] = np.sqrt(np.abs(Rc * Zs) + np.abs(Rs * Zc))`
where `Rc` is the `M=+1` mode and `Rs` is the `M=-1` mode, etc. 

That should always works unless if like `Rc` and `Zc` are both 0 for some reason, which is unlikely. Otherwise we might as well always compute the actual minor radius and all the other quantities.

Reply:
the reason for this PR (@rahulgaur104  correct me if I am wrong) was a "a" which was zero for a stellarator symmetric case, so it did not even have `Rs` or `Zc`